ledger/catchpointwriter.go

```diff
@@ -35,26 +35,32 @@ const (
 	// BalancesPerCatchpointFileChunk defines the number of accounts that would be stored in each chunk in the catchpoint file.
 	// note that the last chunk would typically be less than this number.
 	BalancesPerCatchpointFileChunk = 512
+
+	// DefaultMaxResourcesPerChunk defines the max number of resources that go in a singular chunk
+	// 3000000 resources * 300B/resource => roughly max 1GB per chunk
+	DefaultMaxResourcesPerChunk = 3000000
```

Comment:
1GB is a lot of RAM to handle a chunk, plus it's probably more with various copying/decoding activity going on... plus there are really big resources (app data) that can be greater than 300B. how about e.g. 10x or 100x smaller?

Reply:
You could also be tracking the size of the encoded resources, e.g. resourceBytesCount, but maybe just setting this to 10000 or something would be easier.

Reply:
changed to 10x smaller